support/src/main/java/se/inera/intyg/common/support/facade/model/config/VisualAcuity.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.support.facade.model.config;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.Builder;
+import lombok.Value;
+import se.inera.intyg.common.support.facade.model.config.VisualAcuity.VisualAcuityBuilder;
+
+@JsonDeserialize(builder = VisualAcuityBuilder.class)
+@Value
+@Builder
+public class VisualAcuity {
```

Comment:
There are two things to consider in the config:
1) What precision is the visualAcuity value i.e. how many decimals.
2) What are the labels that represent contact lenses true/false i.e. in this case Ja/Nej.

Both can be handled in the configuration or they could be hardcoded on the client. As this is a custom tool for VisualAcuity, I think it is OK to let it be hardcoded. What are your thoughts?

Reply:
After discussion we decided to keep the original implementation